relayer/processor/metrics.go

```diff
@@ -56,6 +58,14 @@ func (m *PrometheusMetrics) IncTxFailure(pathName, chain, errDesc string) {
 	m.TxFailureError.WithLabelValues(pathName, chain, errDesc).Inc()
 }
 
+func (m *PrometheusMetrics) SetUnrelayedPackets(pathName, srcChain, destChain, srcChannel, destChannel string, unrelayedPackets int) {
+	m.UnrelayedPackets.WithLabelValues(pathName, srcChain, destChain, srcChannel, destChannel).Set(float64(unrelayedPackets))
+}
+
+func (m *PrometheusMetrics) SetUnrelayedAcks(pathName, srcChain, destChain, srcChannel, destChannel string, unrelayedPackets int) {
+	m.UnrelayedAcks.WithLabelValues(pathName, srcChain, destChain, srcChannel, destChannel).Set(float64(unrelayedPackets))
```

Comment:
I noticed both L61/62 and L65/66 reference `unrelayedPackets`

e.g.
```go
.WithLabelValues(pathName, srcChain, destChain, srcChannel, destChannel).Set(float64(unrelayedPackets))
```

in L65 & L66 should `unrelayedPackets` be `unrelayedAcks`?

Reply:
Great catch! Thank you 🙏 
Functionality wise this shouldn't matter but readability wise, this is the way.